pages/profile.tsx

```diff
@@ -193,13 +201,21 @@ export default createPage({
               }}
             />
             <label className="form-check-label" htmlFor="flexCheckChecked">
-              Allow others to see my profile&nbsp;
+              Allow others to see my profile
             </label>
             {profile.updatingIsPublic ? (
               <Spinner animation="border" className="mx-auto" size="sm" />
             ) : null}
           </div>
         )}
+        <div className="mt-2">
+          <Button
```

Comment:
Let's use that `<Wrap><Button/><Wrap>` way to link with buttons, as in the previous PR.